src/misclibs.F

```diff
@@ -1453,30 +1388,18 @@ subroutine pdefqtest(rmax,asq,ruh,rvh,rmh,rho,q3d,dum2d1,dum2d2,dum2d3,dum2d4,du
         enddo
         !$acc end parallel
 
-#ifdef _B4B02R
-        !$acc update host(dum2d5,dum2d2) 
-#else
         !$acc parallel loop vector collapse(2) default(present) private(i,j) reduction(+:asq)
-#endif
         do j=1,nj
         do i=1,ni
           asq=asq+(dum2d5(i,j)-dum2d2(i,j))*tem
         enddo
         enddo
-#ifdef _B4B02R
-        !$acc update device(asq)
-#endif
 
       ELSE
 
-        !FIXME
         !$omp parallel do default(shared) private(i,j,k,a1,a2)
-#ifdef _B4B05R
-        !$acc update host(rho,q3d,ruh,rvh,rmh)
-#else
-        !$acc parallel default(present) private(k) reduction(+:budt) 
+        !$acc parallel default(present) private(k) reduction(+:budt,asq) 
         !$acc loop gang 
```

Comment:
Do we need to add `reduction(+:asq)` here?

Reply:
Good catch.

Reply:
Done